core/src/filed/fd_plugins.cc

```diff
@@ -1835,17 +1831,45 @@ static int MyPluginBopen(BareosWinFilePacket* bfd,
     errno = io.io_errno;
     bfd->lerror = io.lerror;
   }
-
-  Dmsg1(debuglevel, "Return from plugin open status=%d\n", io.status);
-
-  return io.status;
+  //  The plugin has two options for the read/write during the IO_OPEN call:
+  //  1.: - Set io.status to IoStatus::do_io_in_core, and
+  //      - Set the io.filedes to the filedescriptor of the file that was
+  //        opened in the plugin. In this case the core code will read from
+  //        write to that filedescriptor during backup and restore.
+  //  2.: - Set io.status to IoStatus::success, and
```

Comment:
was this truncated? At least it doesn't make sense to me.

Reply:
Yes, this was updated.